ocs_ci/deployment/fusion_data_foundation.py

```diff
@@ -137,3 +142,45 @@ def fusion_service_instance_health_check():
     install_percent = instance_status["installStatus"]["progressPercentage"]
     assert service_health == "Healthy"
     assert install_percent == 100
+
+
+def setup_fdf_pre_release_deployment():
+    """
+    Perform steps to prepare for a Pre-release deployment of FDF.
+    """
+    fdf_image_tag = config.DEPLOYMENT.get("fdf_image_tag")
+    fdf_catalog_name = "isf-data-foundation-catalog"
```

Comment:
Looks like we already have [Fusion catalog name in defaults](https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/ocs/defaults.py#L56), so we can maybe use it from there.
```suggestion
    fdf_catalog_name = defaults.FUSION_CATALOG_NAME
```

Reply:
Fixed!